readthedocs/subscriptions/forms.py

```diff
@@ -1,26 +1,31 @@
 """Subscriptions forms."""
 
 from django import forms
-from django.urls import reverse
 from django.utils.translation import gettext_lazy as _
+from djstripe import models as djstripe
 
-from readthedocs.subscriptions.models import Plan
+from readthedocs.subscriptions.products import get_listed_products
 
 
 class PlanForm(forms.Form):
 
     """Form to create a subscription after the previous one has ended."""
 
-    plan = forms.ChoiceField()
+    price = forms.ChoiceField()
```

Comment:
Are we showing `price` to the user instead of `plan` here?

Reply:
Hmm, yeah, I should override the label name.